src/main/java/org/isf/generaldata/GeneralData.java

```diff
@@ -94,6 +94,8 @@ public final class GeneralData extends ConfigurationProperties {
 
 	public static boolean STRONGPASSWORD;
 	public static int STRONGLENGTH;
+	public static int PASSWORDTRIES;
+	public static int PASSWORDLOCKTIME;
```

Comment:
These new parameters should be added also in settings.properties.dist in "gui"

Reply:
Done; thanks